src/main/java/com/hwr/hwrzeiterfassung/database/controller/LoginController.java

```diff
@@ -0,0 +1,40 @@
+package com.hwr.hwrzeiterfassung.database.controller;
+
+import com.hwr.hwrzeiterfassung.database.repositorys.LoginRepository;
+import com.hwr.hwrzeiterfassung.database.tables.Login;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/login")
+public class LoginController {
+    @Autowired
+    private LoginRepository loginRepository;
+
+    @GetMapping("/basicLogin")
+    public ResponseEntity<HttpStatus> basicLogin(@RequestParam String email, @RequestParam String password) {
+        if (validateLoginInformations(email, password)) {
+            return new ResponseEntity<>(HttpStatus.ACCEPTED);
+        } else {
+            return new ResponseEntity<>(HttpStatus.NOT_ACCEPTABLE);
+        }
+    }
+
+
+    private boolean validateLoginInformations(String email, String password) {
+        Optional<Login> login = loginRepository.findById(email);
+        if (login.isEmpty()) {
```

Comment:
I've looked in the `Optional<T>` documentation and I did not find a method `isEmpty()`, instead I only found `public boolean isPresent()`. If this works, why?

Reply:
`isEmpty()` is normal for check if an Object is Empty or not and work also with` Optional<T>`

Reply:
Ok, that was me being dumb. Thanks :-)